go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/abema/go-mp4
+
+go 1.14
```

Comment:
The version is based on `- image: circleci/golang:1.14` in `.circleci/config.yml`